CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ option(ENABLE_UNDEFINED_SANITIZER "Use UB sanitizer for Debug build" OFF)
 option(ENABLE_TESTS "Enable Valhalla tests" ON)
 option(ENABLE_WERROR "Convert compiler warnings to errors. Requires ENABLE_COMPILER_WARNINGS=ON to take effect" OFF)
 option(ENABLE_BENCHMARKS "Enable microbenchmarking" ON)
+option(INSTALL_VENDORED_HEADERS "Install vendored public headers to <install_dir>/valhalla/third_party" ON)
```

Comment:
I'm open to remove this, but I thought it might be nice to give the option to opt out.